src/styles-new/_controls.scss

```diff
@@ -237,6 +237,10 @@ select {
     border: none;
     border-radius: $controlCr;
     padding: 1px 20px 1px $interiorMargin;
+    *,
+    option {
+        color: #000 !important; // Fix fg color of option elements for Windows
```

Comment:
Hi @charlesh88 
1. Can we use the scss variable instead of hardcoding #000?
2. Can we also write this with correct CSS specificity? Writing  `!important` is not a good practice.

Reply:
Thanks for the input. 

1. No, this is an override on a system-based display that is very browser and system-specific. I always want it to be black.
1. See 1 above - I am hedging my bets here with !important.